tests/test_closing.py

```diff
@@ -3631,3 +3631,41 @@ def test_close_weight_estimate(node_factory, bitcoind):
     signed_weight = int(bitcoind.rpc.decoderawtransaction(tx)['weight'])
     assert signed_weight <= actual_weight
     assert signed_weight >= actual_weight - 2
+
+
+@pytest.mark.developer("needs dev_disconnect")
+def test_onchain_close_upstream(node_factory, bitcoind, executor):
+    """We send an HTLC, and peer unilaterally closes: do we close
+upstream?  Aka
+https://github.com/ElementsProject/lightning/issues/4649
```

Comment:
Nit: Indentation and first line should be short enough not to wrap, followed by an empty line with the long description.

Reply:
Fixed, thanks!